Automattic/pocket-casts-ios
# Bookmarks: Adds the episode bookmark list page

| 🛫 Depends on: https://github.com/Automattic/pocket-casts-ios/pull/1012 |
|:---:|

This adds the ability to swipe to the bookmarks page and displaying of the bookmarks for the specific episode. This doesn't quite match the designs yet, I'll be updating this later.

## Demo Video

https://github.com/Automattic/pocket-casts-ios/assets/793774/bf794d09-858b-473b-b4e9-23796dfa232f


## To test

1. Launch the app
2. Enable the Bookmarks feature flag
3. Go to the Podcasts tab
4. Open a podcast that has bookmarks
5. Tap on an episode that has bookmarks
6. ✅ Verify the episode details load, and you can swipe to see all the show notes
7. ✅ Verify there are no auto layout warnings
8. Swipe to the left
9. ✅ Verify the bookmarks view appears and displays the episodes bookmarks
10. Swipe back to the episode details
11. Swipe down until the shadow appears under the nav bar
12. Swipe left to show the bookmarks
13. ✅ Verify the shadow is not visible
14. Swipe right to show the episode details again
15. ✅ Verify the shadow appears again

## Checklist
